canonical/lxd-ui
# feat(instance) add boot config keys for instances and profiles

## Done

- feat(instance) add boot config keys for instances and profiles

Fixes #994 

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @mas-who or @edlerd for access.
    - With a local copy of this branch, [build and run as described in the docs](../CONTRIBUTING.md#setting-up-for-development).
2. Perform the following QA steps:
    - create an instance with boot configuration in the create form selected
    - edit the instance, ensure the values from creation appear and can be updated
    - create a profile with boot configuration in the create form selected
    - apply the profile to an instance, ensure the inherited values match the profile selected

## Screenshots
![image](https://github.com/user-attachments/assets/ecd3f1df-2193-46dd-8de4-4c257556b86a)
